docs/source/azure/configuration/hostingoptions.rst

```diff
@@ -7,34 +7,36 @@ Overview
 ########
 
 The Azure provider supports deploying policies into Azure Functions to allow
-them to run inexpensively in your subscription.
+them to run inexpensively in your subscription. Currently you can deploy timer 
+triggered functions (azure-periodic) or Event Grid triggered functions 
+(azure-event-grid) depending on your needs.
 
-Python support in Azure Functions V2 is in preview and this feature is still immature.
+The created Azure Functions use the V2 runtime for Python to run Custodian.
+This feature is still in preview and puts the following requirements on the deployment:
 
 - Linux is currently the only supported operating system.
-- Python 3.6+ is the only supported version.
+- Python 3.6.X is the only supported version.
```

Comment:
@logachev I can't find any documentation saying that python 3.7 is supported by Azure functions yet. There are a few github issues out about it, but they all remain open. I was following the [requirements for azure function in this doc](https://docs.microsoft.com/en-us/azure/azure-functions/tutorial-vs-code-serverless-python#visual-studio-code-python-and-the-azure-functions-extension)

Reply:
I changed this from a statement about the limits of the app service plan to a statement of what will be created in the Azure subscription. This way the restrictions around python functions does not matter, because we clearly say what type of app service plan is created